src/FffGcodeWriter.cpp

```diff
@@ -817,6 +817,17 @@ void FffGcodeWriter::addMeshLayerToGCode(SliceDataStorage& storage, SliceMeshSto
     }
     part_order_optimizer.optimize();
 
+    EFillMethod infill_pattern = mesh->getSettingAsFillMethod("infill_pattern");
+    if ((infill_pattern == EFillMethod::LINES || infill_pattern == EFillMethod::ZIG_ZAG) && mesh->infill_angles.size() == 0)
```

Comment:
Can we please clarify what should be done here. Are you still requesting that the infill angles should be able to be specified for any infill type and not just lines+zigzag?